index.html

```diff
@@ -145,7 +147,7 @@ <h2 class="mb-lg font1">Wall Decors</h2>
           <li class="slide shadow-box">
             <div class="grid-2-cols-layout">
               <div class="left-pane">
-                <img src="assets/toy1.jpg" alt="img" class="responsive-img" />
+                <img src="assets/toy1.jpg" alt="img" class="img-responsive" />
```

Comment:
Here you should have a better alt name instead of img, because its always given that its an image right? Having a better alt name tells what that image was supposed to be, like toys, a person, wall decor.